package.json

```diff
@@ -13,6 +13,9 @@
     "build:server": "cross-env NODE_ENV=production webpack --config webpack/config.server.js",
     "build:examples": "cross-env NODE_ENV=production webpack --config webpack/config.examples.js",
     "test": "jest",
+    "xxtest": "npm run test:server && npm run test:client",
```

Comment:
Is this the old command to run the tests? Should this be removed?

Reply:
Ah, thanks so much for spotting them! This was an experiment before I found out about Jest projects.

I've removed them!